lib/cdo/pegasus.rb

```diff
@@ -7,7 +7,6 @@
 require_relative './pegasus/hash'
 require_relative './pegasus/object'
 require_relative './pegasus/string'
-require_relative './pegasus/only_one_running'
```

Comment:
Ideally, not, as I don't consider `only_one` as being part of the pegasus environment. Thus my (current) comment in the PR summary.